Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -66,13 +68,19 @@ class NetworkResponseHandler {
     ///   - requestId: request id associated with current response
     func processResponseOnSuccess(jsonResponse: String, requestId: String) {
         guard let data = jsonResponse.data(using: .utf8) else { return }
+        var ignoreStorePayloads = false
+        if let firstEvent = sentEventsWaitingResponse[requestId]?.first, let lastResetDate = lastResetDate {
```

Comment:
hmm.. this is an interesting one.. we should probably add a comment when batching saying that `multiple events cannot be batched if there was a reset in between them`